lib/clientv2/globalnetworkpolicy.go

```diff
@@ -41,8 +42,16 @@ type globalnetworkpolicies struct {
 // representation of the GlobalNetworkPolicy, and an error, if there is any.
 func (r globalnetworkpolicies) Create(ctx context.Context, res *apiv2.GlobalNetworkPolicy, opts options.SetOptions) (*apiv2.GlobalNetworkPolicy, error) {
 	defaultPolicyTypesField(&res.Spec)
+
+	// Properly prefix the name
```

Comment:
I think we should have converter methods in both directions.

We are disallowing tier names with a kip prefix - so you can use the same check in both directions to add or remove the prefix.

Reply:
Oh just realized - we only need this for NetworkPolicies not GlobalNetworkPolicies